src/components/Chip/Chip.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC, ReactNode, useMemo } from 'react'
+import { VariantProps, tv } from 'tailwind-variants'
+
+export type Props = {
```

Comment:
`React.CompnentProps<typeof Hoge>` のような呼び出しで型は作れるので、基本的に `export` は不要と考えています。

```suggestion
type Props = {
```